packages/client/modules/meeting/components/MeetingCheckInPrompt/NewCheckInQuestion.tsx

```diff
@@ -37,30 +37,6 @@ const CogIcon = styled('div')({
   width: 24
 })
 
-const QuestionBlock = styled('div')({
```

Comment:
While in the area, I changed this to tailwind. It's not directly relevant to the icebreaker styles